src/core/graph.test.js

```diff
@@ -1663,4 +1664,87 @@ describe("core/graph", () => {
       expect(edgeToParts(edge)).toEqual(expected);
     });
   });
+
+  describe("compareGraphs", () => {
```

Comment:
it'd be good to have a test verifying that two graphs which are the same except for a single dangling edge still produces diffs occasionally. (i.e. if someone were to go and swap `showDangling: false` the test should fail)

Reply:
done